doc/manual/types.rst

```diff
@@ -1003,10 +1003,15 @@ which denotes any number of trailing elements:
     julia> isa(("1",1,2,3.0), Tuple{AbstractString,Vararg{Int}})
     false
 
-Notice that ``Vararg{T}`` matches zero or more elements of type ``T``.
+Notice that ``Vararg{T}`` corresponds to zero or more elements of type ``T``.
 Vararg tuple types are used to represent the arguments accepted by varargs
 methods (see :ref:`man-varargs-functions`).
 
+The type ``Vararg{T,N}`` corresponds to exactly ``N`` elements of ``T``.  ``NTuple{N,T}`` is
```

Comment:
if `Vararg{T}` uses the wording "element of type `T`" then shouldn't `Vararg{T,N}` as well?